backend/model/stock.py

```diff
@@ -0,0 +1,22 @@
+import os
+from matplotlib.style import use
+import psycopg2
+
+class StockDAO:
+
+     def __init__(self):
+        self.conn = psycopg2.connect(os.environ["DATABASE_URL"], sslmode="require")
+
+   
+#####################################################################################################
+#                                       CRUD OPERATIONS                                             #
+#####################################################################################################
+
+def getUserSavedStocks(self, userid):
+    cursor = self.conn.cursor()
+    query = "select symbol from  monitors as M ,  \
+        stock as S where M.userid = %s and s.stockid = M.stockid;"
```

Comment:
Capitalize s from s.stockid, just for consistency and better understanding, I know it isn't actually necessary.

Reply:
Just an observation, the query could have been done using natural inner join but is up to you how you implement it, the current implementation works just fine.